mkdocs.yml

```
@@ -90,6 +90,14 @@ plugins:
   macros:
       include_yaml:
 #         - 'variables.yml' # Use in markdown as '{{ VAR }}'
+  with-pdf:
+    output_path: '_pdf/Percona Platform.pdf'
+    cover_title: 'Percona Platform documentation'
+    cover_subtitle: latest update
+    author: "Percona Technical Documentation Team"
+    cover_logo: docs/_images/Percona_Logo_Color.png
```

```suggestion
    cover_logo: docs/images/percona-logo.png
```
This file does not exist. Try to see which logo we want to use from `docs/images/` as there are plenty. Most probably it is `percona-logo.png`.